rspotify-model/Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 readme = "../README.md"
 
 [dependencies]
-chrono = { version = "0.4.19", features = ["serde", "rustc-serialize"] }
+chrono = { version = "0.4.19", features = ["serde","wasmbind"] }
```

Comment:
AFAIK this enables `wasmbind` *always*, even when it's not compiling for `wasm32`. Perhaps it'd be better to use one of these `cfg` statements to only add the feature for `wasm32`?